commonknowledge/meep-intelligence-hub
# Encrypt API keys stored in database

## Description
This PR adds encryption to the Airtable and Mailchimp API keys as they are stored in the database 

## Motivation and Context
Addresses issue [MEEP-268](https://linear.app/commonknowledge/issue/MEEP-268/encrypt-api-key-columns)

## How Can It Be Tested?
- Download the branch and run locally
- Run poetry install to install [cryptography](https://cryptography.io/en/latest/installation/) package
- Generate a value for ENCRYPTION_SECRET_KEY variable. For Fernet, the key must be 32 bytes long. 
- Try adding a Mailchimp and Airtable Data Source
- Observe that the api keys are encrypted in the api_key column in the hub_airtabledatasource and hub_mailchimpdatasource tables in the database

## How Will This Be Deployed?
1. Back up production database before deployment
2. Add ENCRYPTION_SECRET_KEY value to Render

## Types of changes
